README.md

```diff
@@ -58,21 +64,21 @@ To deploy workloads with SR-IOV VF or PCI PF, this plugin needs to work together
     - During Pod creation, plumbs the allocated network device to the Pods network namespace using device information given by the meta plugin
     - On Pod deletion, reset and release the allocated network device from the Pod
 
-
 Please follow the [Quick Start](#quick-start) for multi network interface support in Kubernetes.
 
 ### Supported SR-IOV NICs
 
 The following  NICs were tested with this implementation. However, other SR-IOV capable NICs should work as well.
- - Intel® E800 Series
- - Intel® X700 Series
- - Intel® 82599ES
- - Mellanox ConnectX-4®
- - Mellanox Connectx-4® Lx EN Adapter
- - Mellanox ConnectX-5®
- - Mellanox ConnectX-5® Ex
- - Mellanox ConnectX-6®
- - Mellanox ConnectX-6® Dx
+
+- Intel® Ethernet 800 Series (E810)
+- Intel® Ethernet 700 Series (XL710, X710, XXV710, X72
```

Comment:
nit:

```suggestion
- Intel® Ethernet 700 Series (XL710, X710, XXV710)
```

X722 seems to to not be supported yet:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/templates/configmap.yaml#L6

Reply:
I am enquiring about support for X722. I have removed it for now from the README but will create a new PR to add it if it is in fact supported.

Reply:
Thanks